script.js

```diff
@@ -0,0 +1,47 @@
+import { firstArr } from "./data/4x4.js";
+import { secondArr } from "./data/32x32.js";
+function firstPrint(){
+  let ctx = document.getElementById("canvas").getContext("2d");
+  ctx.clearRect(0,0,512,512);
+  const scale = 128;
+  for (let i = 0; i < 4; i++) {
+    for (let j = 0; j < 4; j++) {
+      ctx.fillStyle = "#" + firstArr[i][j];
+      ctx.fillRect(i*scale,j*scale, (i+1)*scale, (j+1)*scale);
+    }
+  }
+}
+
+function secondPrint(){
+  let ctx = document.getElementById("canvas").getContext("2d");
+  ctx.clearRect(0,0,512,512);
+  const scale = 16;
+  for (let i = 0; i < 32; i++) {
+    for (let j = 0; j < 32; j++) {
+      ctx.fillStyle = "rgba(" + secondArr[i][j][0] + "," + secondArr[i][j][1] + "," + secondArr[i][j][2] + "," + secondArr[i][j][3]/255 + ")";
```

Comment:
Please pay attention to "string literal" feature of es6, it's much more handy than string concatenation